pkg/diagnostics/collectors.go

```diff
@@ -171,6 +174,28 @@ func (c *collectorFactory) PackagesCollectors() []*Collect {
 	return collectors
 }
 
+func (c *collectorFactory) FileCollectors(paths []string) []*Collect {
+	collectors := []*Collect{}
+
+	for _, path := range paths {
+		r := files.NewReader()
+		content, err := r.ReadFile(path)
+
+		if err != nil {
+			logger.V(5).Info(fmt.Sprintf("skipping collection for file %s", path))
```

Comment:
shouldn't we return an error?

Reply:
Well that depends on how you handle the error up the stack
Either way, just logging this won't give any clue to the support engineer as of why the file is missing from the bundle, since this log won't be included in the bundle. Maybe we can make the content of the file the error message?

Reply:
Good idea. I'll work that it